.github/workflows/vscode.yml

```
@@ -78,4 +78,4 @@ jobs:
     needs: quint-vscode-plugin
     runs-on: ubuntu-latest
     steps:
-      - run: echo "All OS tests completed successfully!"
+      - run: echo "All OS tests completed successfully!
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC1009:info:1:1: The mentioned syntax error was in this simple command

(shellcheck)